Makefile

```diff
@@ -6,8 +6,10 @@ else ifeq ($(GOARCH),arm64)
 LINUX_ARCH = aarch64
 endif
 
-GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
-TAG_VERSION ?= $(shell git tag --points-at | sort -Vr | head -n1 | cut -c2-)
+ifeq ($(shell git status -s),)
+	TAG_VERSION ?= $(shell git tag --points-at | sort -Vr | head -n1 | cut -c2-)
```

Comment:
there is also a `git describe --tags` command which creates a tag/sha/dirty-aware version string

may or may not be useful for your purposes

Reply:
Good to know, but it seems to just point to the last commit then, which means uncommitted changes still get the identical tag to the latest commit, so not _quite_ useful for this case at the moment.